java/gui/src/test/java/com/squareup/subzero/wallet/WalletLoaderTest.java

```diff
@@ -105,6 +106,12 @@ public void load() throws Exception {
     assertThat(roundtrip).isEqualTo(wallet);
   }
 
+  @Test
+  public void walletLoaderBadFilePaths() throws Exception {
+    assertThatThrownBy(() -> new WalletLoader("\0"))
+        .isInstanceOf(IllegalArgumentException.class);
+  }
+
```

Comment:
super nit: should we also add the following test cases?
- path looks well-formed but doesn't exist
- path refers to a regular file rather than a directory

Reply:
i didn't include these because they get handled by the marker file check, but it doesn't hurt to have the tests explicitly in case something changes later.